.vscode/launch.json

```diff
@@ -0,0 +1,42 @@
+{
+  "configurations": [
+      {
+          "name": "micro:bit cortex debug",
+          "cwd": "${workspaceFolder}",
+          "executable": "build/MICROBIT",
+          "request": "launch",
+          "type": "cortex-debug",
+          "servertype": "pyocd",
+          "interface": "swd",
+          "device": "nrf52",
+          "targetId": "nrf52",
+          "boardId": "9904",
+          "svdFile": "libraries/codal-nrf52/nrfx/mdk/nrf52833.svd",
+          "preLaunchCommands": [
+            "load build/MICROBIT",
+            "enable breakpoint",
+            "monitor reset"
+          ]
+      },
+
+      {
+          "name": "micro:bit openOCD cortex debug",
+          "cwd": "${workspaceFolder}",
+          "executable": "build/MICROBIT",
+          "request": "launch",
+          "type": "cortex-debug",
+          "servertype": "openocd",
+          "configFiles": [
+			      "interface/cmsis-dap.cfg",
+			      "target/nrf52.cfg"
+		      ],
+          "interface": "swd",
+          "runToMain": false,

```

Comment:
Looks like it might also be obsolete: https://github.com/Marus/cortex-debug/blob/35f42186726063bcb150f2e4132556a7528eea60/package.json#L934

Reply:
Obsolete in this case meant replaced by runToEntryPoint, in case you wanted to preserve this configuration.